.github/workflows/ci.yml

```
@@ -48,6 +48,31 @@ jobs:
         if: ${{ matrix.buildType == 'Release' }}
         run: |
           time ./build/run-test262 -m -c test262.conf -a
+  linux-examples:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: build
+        run: |
+          mkdir build
+          cd build
+          cmake -DBUILD_EXAMPLES=ON -DCMAKE_VERBOSE_MAKEFILE=ON ..
+          cd ..
+          cmake --build build -j$(getconf _NPROCESSORS_ONLN)
+      - name: test
+        run: |
+          ldd build/hello
+          ldd build/hello_module
+          ldd build/test_fib
+          ./build/hello
+          ./build/hello_module
+          ./build/test_fib
+          cp build/fib.so examples/
+          ./build/qjs examples/test_fib.js
+          cp build/point.so examples/
+          ./build/qjs examples/test_point.js
+          cp build/bjson.so tests/
+          ./build/qjs tests/test_bjson.js
```

The most minor of nits but IMO it's a little easier to read when written like this (edit: the diff itself isn't all that legible, I'm afraid):
```suggestion
          cp build/fib.so examples/
          cp build/point.so examples/
          cp build/bjson.so tests/
          ./build/qjs examples/test_fib.js
          ./build/qjs examples/test_point.js
          ./build/qjs tests/test_bjson.js
```
Alternatively, you could use LD_LIBRARY_PATH/DYLD_LIBRARY_PATH so you don't have to copy the files.